Sealnote/src/main/java/com/twistedplane/sealnote/NoteActivity.java

```diff
@@ -107,46 +107,51 @@ public void afterTextChanged(Editable editable) {
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_note);
-        // Even though we change content view later, we secure window as soon as possible
-        Misc.secureWindow(NoteActivity.this);
 
-        mBackgroundColor = 0;
-        mAutoSaveEnabled = PreferenceHandler.isAutosaveEnabled(NoteActivity.this);
+        if(SealnoteApplication.getDatabase().getPassword() == null){
```

Comment:
(1) You can avoid the wrapping rest of the code inside else by writing code like this. This also avoids making the diff bigger.

```
if (cond) {
  stms ...
  return;
}

else-stms...
```

(2) Since this is used twice, maybe you can abstract it as a function like "isPasswordLoaded".or something like that